include/boost/geometry/algorithms/densify.hpp

```diff
@@ -78,11 +78,8 @@ struct densify_range
     static inline void apply(FwdRng const& rng, MutRng & rng_out,
                              T const& len, Strategies const& strategies)
     {
-        typedef typename boost::range_iterator<FwdRng const>::type iterator_t;
-        typedef typename boost::range_value<FwdRng>::type point_t;
-
-        iterator_t it = boost::begin(rng);
-        iterator_t end = boost::end(rng);
+        auto it = boost::begin(rng);
+        auto end = boost::end(rng);
```

Comment:
You see it occasionally, but usually iterators are never declared const.
I would not make them `auto const&` because it's not a reference (though it's syntactically correct)